src/index.html

```diff
@@ -14,7 +14,7 @@
         <link rel="icon" type="image/png" href="./assets/images/MEDIBLOC_Symbol.png" />
         <script>
           /* Following comment is a placeholder used by `npm run build:testnet`, do not remove */
-          var defaultNet = "mainnet"; //testnet
+          var defaultNet = "mainnet";//defaultNetwork
```

Comment:
I found that this line is a placeholder which is replaced by `npm run build:testnet`, but it didn't work.

I made it work for now, but I will make it better in the future.